File: app/src/main/java/ru/ushell/app/ui/SavingSession/Session.java
package ru.ushell.app.ui.SavingSession;

import android.content.Context;
import android.content.SharedPreferences;

public class Session {
    public static void save(Context context, String value, String key) {
        SharedPreferences sharedPreferences = context.getSharedPreferences("clipcode", Context.MODE_PRIVATE);
        SharedPreferences.Editor editor = sharedPreferences.edit();
        editor.putString(key, value);
        editor.apply();

    }

    public static String get(Context context, String key) {
        SharedPreferences sharedPreferences = context.getSharedPreferences("clipcode", Context.MODE_PRIVATE);
        return sharedPreferences.getString(key, "false");
    }
}

File: app/src/main/java/ru/ushell/app/data/condition/session/Session.kt
package ru.ushell.app.data.condition.session

import android.content.Context
import android.content.SharedPreferences
import androidx.core.content.edit

class Session {
    companion object {

        private const val CONDITION = "condition"

        fun save(context: Context, value: String, key: String){
            val sharedPreferences: SharedPreferences =
                context.getSharedPreferences(CONDITION, Context.MODE_PRIVATE)

            sharedPreferences.edit {
                putString(key, value)
            }
        }

        fun get(context: Context, key: String): String? {
            val sharedPreferences: SharedPreferences =
                context.getSharedPreferences(CONDITION, Context.MODE_PRIVATE)

            return sharedPreferences.getString(key, "false")
        }
    }
}